### PR TITLE
DOC-2249: Added v22.1.2 release note

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-2249

### CODE REVIEW

File: _includes/releases/v22.1/v22.1.2.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+<h3 id="v22-1-2-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- CSV is now a supported format for changefeeds. This only works with `initial_scan='only'` and does not work with diff/resolved options. [#82355][#82355]
+
+<h3 id="v22-1-2-sql-language-changes">SQL language changes</h3>
+
+- The `bulkio.ingest.sender_concurrency_limit` [cluster setting](../v22.1/cluster-settings.html) can be used to adjust the concurrency at which any one SQL node, across all operations that it is running (e.g., [`RESTORE`s](../v22.1/restore.html), [`IMPORT`s](v22.1/import.html), and [schema changes](../v22.1/online-schema-changes.html)), will send bulk ingest requests to the KV storage layer. [#81789][#81789]
+- The `sql.ttl.range_batch_size` [cluster setting](../v22.1/cluster-settings.html) is deprecated. [#82711][#82711]
+- Row-level TTL now works for multi-tenant clusters. [#82711][#82711]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.2.md:15]: I thought we weren't supposed to refer to `multi-tenant` in release notes.

File: _includes/releases/v22.1/v22.1.2.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- The pgwire `DESCRIBE` command is now supported for use against a cursor created with the `DECLARE` command. This improves compatibility with PostgreSQL and is needed for compatibility with psycopg3 server-side cursors. [#82772][#82772]
+- Fixed an issue where `SHOW BACKUP with privileges` displayed grant statments with incorrect syntax (specifically, without the object type). As an example, previously displayed: GRANT ALL ON status TO j4;  Now displayed: GRANT ALL ON TYPE status TO j4; [#82727][#82727]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.2.md:17]:
```suggestion
- Fixed an issue where `SHOW BACKUP with privileges` displayed grant statments with incorrect syntax (specifically, without the object type). As an example, previously displayed: `GRANT ALL ON status TO j4;`  Now displayed: `GRANT ALL ON TYPE status TO j4;` [#82727][#82727]
```

File: _includes/releases/v22.1/v22.1.2.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Added the `spanconfig.kvsubscriber.update_behind_nanos` metric to track the latency between realtime and the last update handled by the KVSubscriber. This metric can be used to monitor the staleness of a node's view of reconciled spanconfig state. [#82895][#82895]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.2.md:18]:
```suggestion
- Added the `spanconfig.kvsubscriber.update_behind_nanos` metric to track the latency between realtime and the last update handled by the `KVSubscriber`. This metric can be used to monitor the staleness of a node's view of reconciled spanconfig state. [#82895][#82895]
```

Comment (stbof) [reply _includes/releases/v22.1/v22.1.2.md:18]: What is `spanconfig`?

File: _includes/releases/v22.1/v22.1.2.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+<h3 id="v22-1-2-db-console-changes">DB Console changes</h3>
+
+- The time picker component has been improved such that users can use keyboard input to select a time without having to type `" (UTC)"`. [#82495][#82495]
+- The time picker now opens directly to the custom time selection menu when a custom time is already selected. A "Preset Time Ranges" navigation link has been added to go to the preset options from the custom menu. [#82495][#82495]
+
+<h3 id="v22-1-2-bug-fixes">Bug fixes</h3>
+
+- The output of [`SHOW CREATE VIEW`](../v22.1/show-create.html#show-the-create-view-statement-for-a-view) now properly includes the keyword `MATERIALIZED` for materialized views. [#82196][#82196]
+- Fixed the `identity_generation` column in the [`information_schema.columns`](../v22.1/information-schema.html#columns) table so its value is either `BY DEFAULT` or `ALWAYS` (or null). [#82184][#82184]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.2.md:28]:
```suggestion
- Fixed the `identity_generation` column in the [`information_schema.columns`](../v22.1/information-schema.html#columns) table so its value is either `BY DEFAULT`, `ALWAYS`, or `NULL`). [#82184][#82184]
```

File: _includes/releases/v22.1/v22.1.2.md (hunk 1 above)
Comment (nickvigilante) [reply _includes/releases/v22.1/v22.1.2.md:15]: You're right; I completely forgot. Removing.

**Comment by nickvigilante** [comment]
@stbof TFTR!

@nickvigilante Merge on Wednesday, 22 June.

**Pull request merged**